contracts/v2/interfaces/ITokenWrappedBridgeMintable.sol

```diff
@@ -0,0 +1,22 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.28;
+
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
```

Comment:
If `@openzeppelin/contracts/` is set, it will pick the OZ version stored in contracts, now is version 5, a few weeks ago was version 4, as it creates a bit of ambiguity I would recommend to explicity specify the version as it is done in the other contracts like: `@openzeppelin/contracts5/`